src/main/resources/db/migration/V1_37_droped_name_from_judge_table.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE juror_mod.judge
```

Comment:
there is a typo in the filename. -- dropped

Reply:
Updated :)